.taq/config.json

```diff
@@ -5,7 +5,7 @@
     "network": {
         "ghostnet": {
             "label": "ghostnet",
-            "rpcUrl": "https://rpc.ithacanet.teztnets.xyz",
+            "rpcUrl": "https://ghostnet.ecadinfra.com",
```

Comment:
Are you able to deploy with this new URL?

Reply:
I copied the new config.json into my scaffolded project, and it appears to work fine.